.travis.yml

```diff
@@ -19,23 +19,32 @@ stages:
 
 jobs:
   include:
-    - stage: linting
-      name: "pre-commit"
-      before_install:
-      install: pip install pre-commit
-      script: pre-commit run --all --show-diff-on-failure --verbose --color always
-      after_success:
+    # Test separately: database_cleanup
     - stage: test
       env:
-        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT=1 INCLUDE="database_cleanup"
```

Comment:
This shouldn't be needed, and more as the module hasn't been migrated yet.

Reply:
I just copied the existing config, but indeed this should go then. Just for clarity, if we add/remove a rebel module, this should happen via a new `copier update` call, no manual fiddling with the files, right?

Reply:
I would say yes, but @Yajo can confirm

Reply:
just noticed we exclude things in [v14](https://github.com/OCA/server-tools/blob/14.0/.travis.yml#L34) too, so I added that as rebel modules. It's not exactly the same because the former version didn't test those modules at all.

Reply:
> Just for clarity, if we add/remove a rebel module, this should happen via a new `copier update` call, no manual fiddling with the files, right?

That'd be the best yes.